kore/src/Kore/Variables/Binding.hs

```diff
@@ -11,23 +11,30 @@ module Kore.Variables.Binding
     , Binder (..)
     , existsBinder
     , forallBinder
+    , muBinder
+    , nuBinder
     ) where
 
 import           Control.Comonad.Trans.Env
 import qualified Control.Lens as Lens
 import           Data.Monoid
                  ( Any (..) )
 
+import Kore.SubstVar
+       ( SubstVar (..) )
 import Kore.Syntax.Exists
 import Kore.Syntax.Forall
+import Kore.Syntax.Mu
+import Kore.Syntax.Nu
+import Kore.Syntax.SetVariable
 
 {- | @Binding@ defines traversals for patterns with binders.
 
 @traverseBinder@ and @traverseVariable@ should be empty if the pattern is not
 respectively a binder or a variable at the top level.
 
  -}
-class Binding patternType where
+class Show patternType => Binding patternType where
```

Comment:
Why does this gain a `Show` constraint?

Reply:
We were doing some tracing and debugging, and printing stuff from within the generic code wrote for `Bindings` (such as the `substitute` function) definition seemed to be much easier to do this way. 

I can remove it if you suggest, but we might need it again some time.